templates/window-settings.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="../styles/window-main.css">
```

Comment:
I'm about to add a theme switcher in my next push in a few days. I would change this then. May we leave this temporarily?